packages/frontend/src/pages/help-page/components/integrations-page/integrations-page.tsx

```diff
@@ -94,7 +94,7 @@ export const IntegrationsPage = () => {
               browser and provide new functionality.
             </Text>
             <Text>Drag any of the following buttons into your browser's toolbar:</Text>
-            <VStack spacing="1rem" align="flex-start" pl="2rem">
+            <VStack spacing="1rem" align="flex-start" pl={{ base: 'none', sm: '2rem' }} w="full" overflow="scroll">
```

Comment:
I can't find a reference for `full` as a valid width value.. maybe it should be `auto`?

Reply:
Oh I believe that's from chakra sizes https://chakra-ui.com/docs/theming/theme#sizes setting 100%